### DISCUSSION TITLE
Welcome to LeetCode-Solutions Discussions!

### DISCUSSION THREAD

**Discussion by GouravRusiya30** [codedecks-in/LeetCode-Solutions#149] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TUNDA-EMMANUEL** [comment]
I’m looking to collaborate with other content creators

**Comment by TUNDA-EMMANUEL** [comment]
just found it interesting ,am will to learn to more programming
